changelog/items/bugs-fixed/reset-ssh.md

```diff
@@ -0,0 +1,2 @@
+- Reset SSH connection in `portal-setup-following` to fix (frequent) issue with
+  privilege escalation (become: True => `Incorrect become password.`)
```

Comment:
nit: can we put `become` in quotes here, so it's obivous that it's a command? It took me a while to parse this sentence. :)